docs/howto/openstack/nova/new-server.md

```diff
@@ -0,0 +1,339 @@
+# Creating new servers in Cleura Cloud
+
+Once you have an [account in {{extra.brand}} 
+Cloud](/howto/getting-started/create-account), you can create virtual 
+machines ---henceforth simply _servers_--- using either the 
```

Comment:
The convention I've always followed for English is that there is one whitespace on either side of the em dash, thus I believe this should say `--- henceforth simply _servers_ ---`.

Reply:
It's a bit different compared to the convention I follow -- but sure, let's go with the single-whitespace-on-either-side-of-the-em-dash one :)